bootstrap/16-files.sh

```diff
@@ -0,0 +1,9 @@
+if [ ! -z $CUSTOM_FILES ]; then
+    info "Copy ${CUSTOM_FILES} to ${R}"
```

Comment:
Try to follow the style of the project. So, pay attention to using the examples of `info`:
* `info "upgrading chroot environment"`
* `info "exiting since CREATE_ONLY_CHROOT is set to true"`
* `info "installing kernel package"`

I guess it's clear the line should look like: `info "copying ${CUSTOM_FILES} to ${R}"`